google-cloud-storage/src/test/scala/docs/scaladsl/GCStorageSourceSpec.scala

```diff
@@ -289,11 +341,14 @@ class GCStorageSourceSpec
 
     "download file when file exists" in {
       // Ensure file content is above the size limit set by akka.http.client.parsing.max-content-length
-      val fileContent = "Google storage file content" + ("x" * 10000)
+      val fileContent = "Google storage file content" + ("x" * 1000)
```

Comment:
This was a mistake (I didn't read the comment properly), need to revert and fix this test.

Reply:
Fixed.